### PR TITLE
Fix for Taxonomy Importer - removes the file upload field

### LINKED ISSUE #1733: Taxonomy importer is requesting a gff3 file
### Tripal Version

4.x

### Drupal Version

n/a

### PostgreSQL Version

n/a

### PHP Version

n/a

### General Description

If I recall, but I don't really, this may be a simple annotation problem. The taxonomy importer is asking for a gff3 file

### Steps to reproduce

Data Loaders -> Taxonomy Loader

### Error messages

_No response_

### Screenshots

20240108_taxonomy_importer_gff3

### PR DESCRIPTION
**Pull request by risharde**
Thanks to @dsenalik for discovering this issue. Hopefully I have resolved this correctly by removing the file upload field altogether.

### CODE REVIEW

File: tripal_chado/src/Plugin/TripalImporter/TaxonomyImporter.php [hunk 1]
```diff
@@ -23,9 +23,10 @@
  *    use_analysis = False,
  *    require_analysis = False,
  *    button_text = @Translation("Import Taxonomy file"),
```
Comment (dsenalik) [inline tripal_chado/src/Plugin/TripalImporter/TaxonomyImporter.php]: This line should just say "Import Taxonomy"

**Review by dsenalik — CHANGES_REQUESTED** [review]
Only one suggestion, but works perfectly

**Comment by risharde** [comment]
@dsenalik Thanks Doug, I made the change on the text

**Review by dsenalik — APPROVED** [review]
Perfect, we can merge once tests pass

**Pull request merged**